js/src/Components/Widgets/LinkWidget.js

```diff
@@ -351,7 +351,7 @@ export const LinkWidget = ({fieldId, defaultValue, onFieldChange, settings}) =>
               fullWidth
               label="URL"
               variant="outlined"
-              helperText="Start typing the title of a piece of content to select it. You can also enter an internal path such as /foo/bar or an external URL such as http://example.com. Enter <front> to link to the front page."
+              helperText={settings.help + " Start typing the title of a piece of content to select it. You can also enter an internal path such as /foo/bar or an external URL such as http://example.com. Enter <front> to link to the front page."}
```

Comment:
What happens if `settings.help` contains some html?

Reply:
I get the point of this, but the normal Dupal UI has the help text below the link text field. It would be nice to keep it as close to consistent as we can. But I do see the issue with the link field being at the bottom of the form.

Reply:
I noticed it's only an issue on cert screen heights too.

Reply:
I refactored things a bit to still use the `FormHelperText` component and keep it located below the title field.